presentation/src/main/java/com/architect/coders/mu8/utils/PresentationExtensions.kt

```diff
@@ -17,26 +18,25 @@ import androidx.lifecycle.ViewModelProviders
 import com.architect.coders.mu8.R
 import com.bumptech.glide.Glide
 
-fun View.makeItGone() {
-    visibility = View.GONE
+fun Context.toast(message: String) {
```

Comment:
If you have an extension function for toast, can you create one for the snack bar?

Reply:
Done